controller/api/public/grpc_server.go

```diff
@@ -231,22 +223,6 @@ func (s *grpcServer) SelfCheck(ctx context.Context, in *healthcheckPb.SelfCheckR
 	return response, nil
 }
 
-func (s *grpcServer) Config(ctx context.Context, req *pb.Empty) (*configPb.All, error) {
```

Comment:
This API is also removed, as there will be no `linkerd-config` to directly retrieve and give out configuration

Reply:
As this `Config` returns the older format which we do not intend to support, so removing this in the PR should be the right solution here.